src/main/_docs/admin-guide/admin-runbook.md

```diff
@@ -32,22 +31,205 @@ Zookeeper is a key-value store that is needed by Swarm in a clustered Codenvy se
+5. Create swap volume with size 4 GB: `#  lvcreate -L 4G -n swap vg-docker`
+6. Create docker’s metadata volume with size 2 GB: `#  lvcreate -L 2G -n metadata vg-docker`
+7. Create docker data volume using all remaining disk space: `#  lvcreate -l 100%FREE -n data vg-docker`
+8. Make swap volume become swap: `#  mkswap /dev/vg-docker/swap`
+9. Activate swap volume: `#  swapon /dev/vg-docker/swap`
+
+For databases and logs:
+
+1. Add data disk to LVM: `# pvcreate /dev/sdc`
+2. Create volume group: `# vgcreate vg-data /dev/sdc`
+3. Create journal volume with size 10 GB: `#  lvcreate -L 10G -n journal vg-data`
+4. Create logs volume with size 20 GB: `#  lvcreate -L 20G -n logs vg-data`
+5. Create machine-logs volume with size 30 GB: `#  lvcreate -L 30G -n machine-logs vg-data`
+6. Create ldap volume with size 10 GB: `#  lvcreate -L 10G -n ldap vg-data`  (Not needed after 5.0.0-M5)
+7. Create mongo volume with size 20 GB: `#  lvcreate -L 20G -n mongo vg-data`  (Not needed after 5.0.0-M5)
+8. Create pgsql volume with size 20 GB: `#  lvcreate -L 20G -n psql vg-data`
+9. Create docker data volume using all remaining disk space: `#  lvcreate -l 100%FREE -n docker vg-docker`
+
+For filesystem:
+
+1. Add data disk to LVM: `# pvcreate /dev/sdd`
+2. Create volume group: `# vgcreate vg-fs /dev/sdd`
+3. Create FS volume using all remaining disk space: `#  lvcreate -l 100%FREE -n fs vg-fs`
+
+For Docker snapshots:
+
+1. Add data disk to LVM: `# pvcreate /dev/sde`
+2. Create volume group: `# vgcreate vg-ddistr /dev/sde`
+3. Create FS volume using all remaining disk space: `#  lvcreate -l 100%FREE -n ddistr vg-ddistr`
+
+Make XFS filesystem for:
+
+1. journal: `# mkfs.xfs /dev/vg-data/journal`
+2. logs: `# mkfs.xfs /dev/vg-data/logs`
+3. machine-logs: `# mkfs.xfs /dev/vg-data/machine-logs`
+4. ldap: `# mkfs.xfs /dev/vg-data/ldap`  (Not needed after 5.0.0-M5)
+5. mongo: `# mkfs.xfs /dev/vg-data/mongo`  (Not needed after 5.0.0-M5)
+6. pgsql: `# mkfs.xfs /dev/vg-data/pgsql`
+7. docker: `# mkfs.xfs /dev/vg-data/docker`
+8. fs: `# mkfs.xfs /dev/vg-fs/fs`
+9. Snapshots: `# mkfs.xfs /dev/vg-ddistr/ddistr`
+
+For /etc/fstab entries (note the *nofail* option):
+
+```
+#
+# /etc/fstab
+# Created by anaconda on Thu Mar 14 11:03:34 2013
+#
+# Accessible filesystems, by reference, are maintained under '/dev/disk'
+# See man pages fstab(5), findfs(8), mount(8) and/or blkid(8) for more info
+#
+LABEL=rootfs            /           ext4    defaults,relatime  1   1
+tmpfs                   /dev/shm                tmpfs   defaults        0 0
+devpts                  /dev/pts                devpts  gid=5,mode=620  0 0
+sysfs                   /sys                    sysfs   defaults        0 0
+proc                    /proc                   proc    defaults        0 0
+/dev/vg-docker/swap     swap                    swap    defaults,nofail        0 0
+/dev/vg-data/fs         /home/codenvy/codenvy-data/fs   xfs     defaults,nofail         0 0
```

Comment:
This directory doesn't exist.  We need to map these various mounts to the right Docker ones.
Also - is this centos only, or are we sure this will work on all flavors of linux?